pallets/manta-sbt/src/lib.rs

```diff
@@ -885,21 +897,11 @@ where
     ) -> PullResponse {
         let (more_receivers, receivers) =
             Self::pull_receivers(*checkpoint.receiver_index, max_receivers);
-        let mut receivers_total = (0..=255)
-            .map(|i| ShardTrees::<T>::get(i).current_path.leaf_index as u128)
-            .sum::<u128>();
-        // Workaround for the fact that we index the receivers from 0
-        if receivers_total == 0u128 {
-            // The cast is fine because the vector sizes are limited by PULL_MAX_RECEIVER_UPDATE_SIZE
-            receivers_total = receivers.len() as u128;
-        } else {
-            receivers_total += 256u128;
-        }
```

Comment:
where is this part ?

Reply:
i'd leave the logic here, to make it easier on the users.

Reply:
Make sure `pull_ledger_total_count` is consistent with `senders_receivers_total` returned by `pull_ledger_diff`, directly returns the value of `pull_ledger_total_count`

Reply:
Sorry i meant where is the other part `(if total_senders_receivers == 256) total_senders_receivers = receivers.count()`

Reply:
I don’t think it’s necessary to add that logic for a new chain with only a few ledger data. If you think we need to add, then I may need to request `pull_receivers` on the `pull_ledger_total_count` interface to be compatible with the correct data, as I said above.

Reply:
This is what I considered when adding `pull_ledger_total_count`

Reply:
im fine either way, let's see what the others say

Reply:
Looks fine to me